src/core/base-runner.js

```diff
@@ -31,6 +31,7 @@ function toRunnable(plug) {
       try {
         if (typeof plug.default === "function") {
           await plug.default(respecDoc);
+          resolve();
```

Comment:
oops? Was this a long standing bug?

Reply:
Nope, I added `plug.default` thing and just forgot calling this.

Reply:
oh, ok 😁